src/nvim/options.lua

```diff
@@ -2488,6 +2488,12 @@ return {
       varname='p_tbidi',
       defaults={if_true={vi=false}}
     },
+    {
+      full_name='termencoding', abbreviation='tenc',
+      type='string', scope={'global'},
+      vi_def=true,
+      defaults={if_true={vi=""}}
+    },
```

Comment:
Maybe add a comment with it that this is a no-op(tion)?  But the help/doc might be enough in that regard.

Reply:
This is just the standard pattern for a disabled option ( i e something that vim only supports on some platforms). But perhaps a general comment could be made at the top of the file (i e if the p_varname is missing then the option is disabled, and &opt gives the default value).